### ISSUE TITLE
Migration did not update `roam_alias` to `roam_aliases`

### ISSUE THREAD

**Issue by alexkehayias** [org-roam/org-roam#1596]
### Description

After successfully running `org-roam-migrate-wizard` I noticed that each of my note files still have `ROAM_ALIAS` even though tat was replaced by `ROAM_ALIASES` in v2. 

#### Steps to Reproduce
Run `org-roam-migrate-wizard`. 

#### Expected Results

I was expecting `#+ROAM_ALIAS` to be replaced with `#+ROAM_ALIASES`

#### Actual Results

Files after migration still have `#+ROAM_ALIAS`

### Environment

- Emacs: GNU Emacs 27.2 (build 1, aarch64-apple-darwin20.4.0, Carbon Version 164 AppKit 2022.44)
 of 2021-06-04
- Framework: 
- Org: Org mode version 9.4.4 (9.4.4-21-g61336f-elpaplus @ [redacted]/elpa/org-plus-contrib-20210215/)
- Org-roam: 2.0.0

**Comment by jethrokuan** [comment]
Can you provide a minimal Org file that I can use? Feel free to strip any private information.

**Comment by alexkehayias** [comment]
Here's an example note:

Original:

```
#+TITLE: Silurian Hypothesis
#+DATE: 2020-06-27
#+ROAM_ALIAS:
#+ROAM_TAGS:

Posits that there were previous civilizations on Earth before humans. Some scientists are conducting research to see if we could detect changes in things like atmospheric carbon, plastics, and nuclear fallout since it's highly likely no other physical evidence would exist e.g. buildings and artifacts.
```

Migrated:

```
:PROPERTIES:
:ID:       2B14EE93-63E7-4B80-9D28-54373F675C45
:END:
#+TITLE: Silurian Hypothesis
#+DATE: 2020-06-27
#+ROAM_ALIAS:

Posits that there were previous civilizations on Earth before humans. Some scientists are conducting research to see if we could detect changes in things like atmospheric carbon, plastics, and nuclear fallout since it's highly likely no other physical evidence would exist e.g. buildings and artifacts.
```

I'm realizing now that `roam_alias` is meant to be migrated to `roam_aliases` _in the `properties` drawer_. I checked a note that had an alias and it did indeed add `roam_aliases` to the properties drawer so this might just be because `roam_alias` here is empty.

**Comment by jethrokuan** [comment]
Hmm yes that might be confusing, although I'm pretty sure in v1 empty were invalid values for the two keywords. I'll patch the migration wizard to handle the empty values anyway.

**Comment by alexkehayias** [comment]
Thank you, although if they weren't valid values to begin with I would just close this issue. I can find and replace to get all the notes back to a valid state, I just thought it was a bug. 

And thank you @jethrokuan for all of your work on org-roam!

**Comment by jethrokuan** [comment]
Closed with #1608
